app/src/main/java/io/github/trojan_gfw/igniter/MainActivity.java

```diff
@@ -205,9 +178,6 @@ protected void onCreate(Bundle savedInstanceState) {
         clashLink.setMovementMethod(LinkMovementMethod.getInstance());
         startStopButton = findViewById(R.id.startStopButton);
 
-        Globals.Init(this);
```

Comment:
Why this is removed?

Reply:
I call Globals.init in MainInitializer and Tool Initializer, it will be init at IgniterApplication.onCreate()